app/views/comments/_form.html.erb

```diff
@@ -67,9 +67,9 @@
     </div>
 
     <div class="response-templates-container crayons-card crayons-card--secondary p-4 mb-4 comment-form__templates fs-base hidden">
-      <header>
-        <button type="button" class="personal-template-button crayons-btn crayons-btn--outlined active" data-target-type="personal" data-form-id="new_comment">Personal</button>
-        <button type="button" class="moderator-template-button crayons-btn crayons-btn--outlined hidden" data-target-type="moderator" data-form-id="new_comment">Moderator</button>
+      <header class="mb-3">
+        <button type="button" class="personal-template-button active" data-target-type="personal" data-form-id="new_comment">Personal</button>
+        <button type="button" class="moderator-template-button hidden" data-target-type="moderator" data-form-id="new_comment">Moderator</button>
```

Comment:
```suggestion
        <button type="button" class="crayons-btn personal-template-button active" data-target-type="personal" data-form-id="new_comment">Personal</button>
        <button type="button" class="crayons-btn moderator-template-button hidden" data-target-type="moderator" data-form-id="new_comment">Moderator</button>
```

I think you removed too many classes from these in the refactor 😅  Looks like with these added back in, everything looks great 👍

Reply:
Oops missed that 😆

Reply:
No worries - I've done it many times! If you can push up this change I'll re-review (seems to still be missing this class) 👍

Reply:
Hmmm not pushed with my other computer... I edited that with my smartphone, not practice!